project_test/lib/pipeline_configurations.py

```diff
@@ -0,0 +1,88 @@
+from core_data_modules.cleaners import swahili, Codes
+
+from project_test.lib.test_schemes import CodeSchemes
+
+class CodingPlan(object):
+    def __init__ (self, raw_field, coded_field, coda_filename, cleaner=None, code_scheme=None, time_field=None,
+                    run_id_field=None, icr_filename=None, analysis_file_key=None, id_field=None):
+        self.raw_field = raw_field
+        self.coded_field = coded_field
+        self.coda.coda_filename = coda_filename
+        self.icr_filename = icr_filename
+        self.cleaner = cleaner
+        self.code_scheme = code_scheme
+        self.time_field = time_field
+        self.run_id_field = time_field
+        self.analysis_file_key = analysis_file_key
+
+        if id_field is None:
+            id_field = "{}_id".format(self.raw_field)
+        self.run_id_field = id_field
+
+class PipelineConfiguration(object):
+    
+    DEV_MODE = False
```

Comment:
You probably want this to be True for now.

Apologies for there not being a comment saying what it's for - when True, data from test users will be included in the analysis file, which is really useful when you're building pipelines. 
Also, this should probably have been a command line flag...